terraform/azure/bootstrap/main.tf

```diff
@@ -12,7 +12,7 @@ resource "azurerm_resource_group" "resource_group" {
 }
 
 resource "azurerm_storage_account" "remote_state_sa" {
-  name                     = "${lower(var.name)}tfstorage"
+  name                     = "ccpdmedicaltfstorage"
```

Comment:
Would it be better to keep the TF file as is and just use a lower case name in the tfvars?

Reply:
for some reason this was erroring for me so I just hardcoded the string.... not sure if something changed in tf since I originally wrote these. 

In this case, this is just setting up the remote state storage for the rest of the deployment, and this script shouldn't need to run again, so I'll back out the change anyway.